packages/onboarding/src/flow-progress/use-flow-progress.ts

```diff
@@ -61,6 +67,12 @@ const flows: Record< string, { [ step: string ]: number } > = {
 		checkPlan: 4,
 		storeAddress: 5,
 	},
+	[ COPY_SITE_FLOW ]: {
+		intro: 0,
+		siteCreationStep: 1,
+		processing: 1,
```

Comment:
Is it intentional that both `siteCreationStep` and `processing` are `1`?

Reply:
Yes!, it's intentional. It just updates the progress bar so it's based on the time it takes each step.

https://github.com/Automattic/wp-calypso/blob/120a74f3c1f555c207af7f8a77fdddbb4785e827/packages/onboarding/src/flow-progress/use-flow-progress.ts#L59-L69

<img width="246" alt="Screenshot 2023-01-13 at 09 22 01" src="https://user-images.githubusercontent.com/779993/212284369-699a753a-9c9e-47a4-9176-f133098823b2.png">